 SQL_DDL_Func_Nino_Jintcharadze_HW.sql .sql

```diff
@@ -0,0 +1,147 @@
+--Task 1. Create a view
+
+CREATE OR REPLACE VIEW get_sales_revenue_by_category_qtr AS
+WITH current_quarter AS (
+    SELECT 
+        CEIL(EXTRACT(MONTH FROM CURRENT_DATE)::NUMERIC / 3) AS quarter,
+        EXTRACT(YEAR FROM CURRENT_DATE) AS year
+)
+SELECT cat.name,SUM(pay.amount) AS total_sales_revenue
+FROM  category cat
+INNER JOIN film_category fm ON fm.category_id = cat.category_id
+INNER JOIN film f ON f.film_id = fm.film_id
+INNER JOIN inventory i ON i.film_id = f.film_id
+INNER JOIN rental r ON r.inventory_id = i.inventory_id
+INNER JOIN payment pay ON pay.rental_id = r.rental_id
+INNER JOIN current_quarter cq ON EXTRACT(YEAR FROM r.rental_date) = cq.year
+                      AND CEIL(EXTRACT(MONTH FROM r.rental_date)::NUMERIC / 3) = cq.quarter
+GROUP BY cat.name
+HAVING SUM(pay.amount) > 0;
+
+
+SELECT cat.name, SUM(pay.amount) AS total_sales_revenue
+FROM category cat
+INNER JOIN film_category fm ON fm.category_id = cat.category_id
+INNER JOIN film f ON f.film_id = fm.film_id
+INNER JOIN inventory i ON i.film_id = f.film_id
+INNER JOIN rental r ON r.inventory_id = i.inventory_id
+INNER JOIN payment pay ON pay.rental_id = r.rental_id
+WHERE EXTRACT(YEAR FROM r.rental_date) = 2017  -- i have data only for this year
+      AND CEIL(EXTRACT(MONTH FROM r.rental_date)::NUMERIC / 3) = 1  -- Example for the first quarter of 2017
+GROUP BY cat.name
+HAVING SUM(pay.amount) > 0;
+
+--Task 2. Create a query language functions
+CREATE OR REPLACE FUNCTION get_sales_revenue_by_category_qtr(quarter INT, year INT)
```

Comment:
Could you please modify the parameter of function - one parameter representing the current quarter and year ?